packages/coin-vet/src/utils/transactionUtil.ts

```diff
@@ -0,0 +1,98 @@
+import * as stringUtil from './stringUtil';
+import * as types from '../config/types';
+const { sha3_256 } = require('js-sha3');
+const elliptic = require('elliptic');
+const ec = new elliptic.ec('secp256k1');
+import Web3 from 'web3';
+import Web3Utils from 'web3-utils';
+import { safeToLowerCase } from './scriptUtil';
+const fastJsonStableStringify = require('fast-json-stable-stringify')
+
+/**
+ * @description Trim Hex for Address
+ * @param {string} hexString expect 32 bytes address in topics
+ * @return {string} 20 bytes address + "0x" prefixed
+ */
+function trimFirst12Bytes(hexString: string): string {
+  return '0x'.concat(hexString.substr(hexString.length - 40));
+}
+
+/**
+ * Convert public key to address
+ * @param {string} compressedPubkey
+ * @return {string}
+ */
+export function pubKeyToAddress(compressedPubkey: string): string {
+  const keyPair = ec.keyFromPublic(compressedPubkey, 'hex');
+  const pubkey = `0x${keyPair.getPublic(false, 'hex').substr(2)}`;
+  const address = trimFirst12Bytes(Web3.utils.keccak256(pubkey));
+  return Web3.utils.toChecksumAddress(address);
+}
+
+export const getRawTx = (transaction: types.Record): any => {
+  const raw = []
+  const rawData1 = [];
+  rawData1.push(transaction.chainTag);
+  rawData1.push(transaction.blockRef);
```

Comment:
`blockRef` might have leading zero(ex. `0x00ca4fdd68fd2a5e`). Consider removing the leading zero.